docs/content/guides/operator/data-ingestion.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Custom Indexer
+description: You can build custom indexers using the Sui micro-data ingestion framework. To create an indexer, you subscribe to a checkpoint stream with full checkpoint content. Establishing a custom indexer helps improve latency, allows pruning the data of your Sui Full node, and provides efficient assemblage of checkpoint data.
+---
+
+You can build custom indexers using the Sui micro-data ingestion framework. To create an indexer, you subscribe to a checkpoint stream with full checkpoint content. This stream can be one of the publicly available streams from Mysten Labs, one that you set up in your local environment, or a combination of the two.  
+
+![Data ingestion flow](./images/data-ingestion.png)
```

Comment:
I would prefer not to include this version of a diagram(it's a bit mysten specific). If you think it's necessary, I'll redo the diagrams and move it to a different place on this page

Reply:
That would be awesome. I was going to say that I would do it, but you'd make a better one.